src/libs/isReportMessageAttachment.js

```diff
@@ -4,10 +4,14 @@ import CONST from '../CONST';
  * Check whether a report action is Attachment or not.
  * Ignore messages containing [Attachment] as the main content. Attachments are actions with only text as [Attachment].
  *
- * @param {Object} reportActionMessage report action's message as text and html
+ * @param {Object} reportActionMessage report action's message as text, html and translationKey
  * @returns {Boolean}
  */
-export default function isReportMessageAttachment({text, html}) {
+export default function isReportMessageAttachment({text, html, translationKey}) {
+    if (translationKey) {
```

Comment:
👋 This was partially responsible for a regression in https://github.com/Expensify/App/issues/24246
It was mainly a back-end problem, the translation key is not set/reset when we change from a message _with_ a translation key to one _without_, or vice-versa
While this wasn't caused by this PR, leaving a note to myself and others to test more complex test cases (this one included testing with two accounts)